lyncs_quda/dirac.py

```diff
@@ -33,31 +32,45 @@ class Dirac:
     csw: float = 0
     mu: float = 0
     epsilon: float = 0
+    matPCtype: str = "INVALID"
```

Comment:
does this parameter need to be exposed line this?
Maybe better to have `parity` as a parameter here and then set `matPCtype` accordingly?
I guess then you don't need to change the attributes if parity is fixed for the operator

Reply:
there are 4 possibilities (EVEN and ODD with SYMM or ASYMM)